src/quo2/components/selectors/selectors.cljs

```diff
@@ -0,0 +1,129 @@
+(ns quo2.components.selectors.selectors
+  (:require [reagent.core :as reagent]
+            [quo.react :as react]
+            [quo.react-native :as rn]
+            [quo2.foundations.colors :as colors]
+            [cljs-bean.core :as bean]
+            [quo.theme :as theme]
+            [quo2.components.icon :as icons]))
+
+(defn get-color [value disabled?]
+  (if value
+    (colors/theme-colors
+     (colors/custom-color :primary 50 (when disabled? 30))
+     (colors/custom-color :primary 60 (when disabled? 30)))
```

Comment:
[custom-color-by-theme](https://github.com/status-im/status-mobile/blob/b4d95b240d88066c34c0e8fcd966a214ac0b0912/src/quo2/foundations/colors.cljs#L248) can be used